setup.cfg

```diff
@@ -1,5 +1,16 @@
 [flake8]
 max-complexity = 10
+ignore =
+    # Defaults
+    # https://gitlab.com/pycqa/flake8/blob/master/src/flake8/defaults.py
+    E121,
+    E123,
+    E126,
+    E226,
+    E24,
+    E704,
+    W503,
+    W504,
```

Comment:
`flake8` has a list of codes it ignores by default, visible at the link I put in `setup.cfg`. Because we did not explicitly list any codes, we were implicitly using the default list.

Now that we want to ignore the print function error (`T003`) we need to explicitly declare the ignored codes, at least for the test code. I thought it would be better to have the list explicit for the main code as well, so they don't diverge.